jwst-masterclass/0.1.0/kernel_runner.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
```

Comment:
If you activate the correct conda environment in the ENTRYPOINT script, you can ensure that `python3` is always pointing to the correct environment.